03/app.js

```diff
@@ -1,3 +1,19 @@
 export default function randomNumber(min, max) {
-
-}
\ No newline at end of file
+    const randomNumber = Math.floor(Math.random() * (max - min + 1) + min);
+    if (typeof min === 'undefined' || typeof max === 'undefined') {
+        throw new Error('This function should contains two number value [min],[max]');
+    };
+    if (typeof min !== 'number' || typeof max !== 'number') {
```

Comment:
Lepiej sprawdzić, która jest niepoprawna i taki dać komunikat czyli sprawdzenie każdej wartości z osobna.

Reply:
Czyli przez to co rozumiem nie powinienem sprawdzać dwóch wartości na raz w jednym if'ie tylko wynieść każda z osobna?

Reply:
Tak, potwierdzam :)